modules/cert-manager-acme-http01.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-http01_{context}"]
+// = Using the HTTP01 Ingress solver
+
+= Issuing certificates by using the HTTP01 challenge
+// TODO: Name
+
+TODO: intro
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* TODO
+// * User workload to expose
+// TODO: ^ from gdoc, what does this mean?
+
+.Procedure
+
+. Create an ACME cluster issuer.
+
+.. Create a YAML file that defines the `ClusterIssuer` object:
++
+.Example `acme-cluster-issuer.yaml` file
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod                                           <1>
+spec:
+  acme:
+    preferredChain: ""
+    privateKeySecretRef:
+      name: example-issuer-account-key-prod                        <2>
+    server: https://acme-staging-v02.api.letsencrypt.org/directory <3>
+    solvers:
+      - http01:
+          ingress:
+            class: openshift-default                               <4>
+----
+<1> TODO: Need description or know if they need to change.
+<2> TODO: Need description or know if they need to change.
+<3> TODO: Need description or know if they need to change.
+<4> TODO - Specify `openshift-default` to use the Ingress-to-Route controller (for what purpose).
+
+
+.. Create the `ClusterIssuer` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f acme-cluster-issuer.yaml
+----
+
+. Create an ingress to expose the service of the user workload.
+
+.. Create a YAML file that defines the `Ingress` object:
++
+.Example `ingress.yaml` file
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: sample-ingress                                           <1>
+  namespace: sandbox                                             <2>
```

Comment:
@thejasn @swghosh Question like the other file: Is sandbox the ns we want to say, or do you have another one we should put?  And, does this ns already need to exist before we get to this step?

Reply:
Updating per answer in the other file